playground/nextjs-pages-router/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "playground-nextjs-pages-router",
```

I don't want to have to type this al the time 😄 
```suggestion
  "name": "playground-nextjs-pages",
```